src/liblzma/check/crc32_table.c

```diff
@@ -26,9 +26,14 @@
 #	define ARM64_CRC32_NO_TABLE 1
 #endif
 
+#if defined(HAVE_LOONGARCH_CRC32)
+#	define LOONGARCH_CRC32_NO_TABLE 1
+#endif
+
 
 #if !defined(HAVE_ENCODERS) && (defined(X86_CLMUL_NO_TABLE) \
-		|| defined(ARM64_CRC32_NO_TABLE_))
+		|| defined(ARM64_CRC32_NO_TABLE_) \
+		|| defined(LOONGARCH_CRC32_NO_TABLE))
```

Comment:
Thanks for remembering this! I had forgotten this was needed in the first round of review